Misc/NEWS.d/next/Build/2019-02-21-14-48-31.bpo-31904.J82jY2.rst

```diff
@@ -0,0 +1 @@
+Enable cpython build system to cross-build for VxWorks RTOS.
```

Comment:
nitpick:  "Enable build system to cross-build for VxWorks RTOS." (remove "cpython" or at least write it with the correct case: "CPython" ;-))